psydac/fem/tests/test_product.py

```diff
@@ -30,7 +30,7 @@ def test_product_space_2d():
     Vy = TensorFemSpace(domain_decomposition, V1, V2)
     # ...
 
-    V = ProductFemSpace(Vx, Vy)
+    V = create_product_space(Vx, Vy)
```

Comment:
This could be replaced with `VectorFemSpace(Vx, Vy)`, correct?

Reply:
absolutely! I replaced it